dali/operators/reader/coco_reader_op.cc

```diff
@@ -102,12 +102,12 @@ If a file list is not provided, this argument is required.)code",
 
 If set to False, the bboxes are returned as [x, y, width, height].)code",
       false)
+  .AddOptionalArg("include_is_crowd",
```

Comment:
I'd recommend either sticking to the exact name (no underscore) or removing the "is" entirely:

```suggestion
  .AddOptionalArg("include_iscrowd",
```
or
```suggestion
  .AddOptionalArg("include_crowd",
```

Reply:
Fixed